com.unity.multiplayer.mlapi/Tests/Editor/BitSerializerTests.cs

```diff
@@ -4,7 +4,8 @@
 using NUnit.Framework;
 using UnityEngine;
 
-namespace MLAPI.EditorTests
+// TODO: put this back!!!
+/*namespace MLAPI.EditorTests
```

Comment:
this is definitely not acceptable :)
I did this because I can no longer use `var inReader = PooledBitReader.Get(inStream)` API

Reply:
I've fixed this is the new equivalent file (NetworkSerializerTests), although my solution requires a NetworkManager to be present in the test scene.